docs/feature-guides/clarinet-js-sdk.md

```diff
@@ -43,23 +43,21 @@ async function main() {
 main();
 ```
 
-By default, the SDK will look for a Clarinet.toml file in the current working directory.  
-It's also possible to provide the path to the manifest like so:
+By default, the SDK will look for a Clarinet.toml file in the current working directory. It's also possible to provide the path to the manifest like so:
 
 ```ts
 const simnet = await initSimnet("./path/to/Clarinet.toml");
 ```
 
-## API references
+## API References
 
 ### `initSimnet`
 
 ```ts
 initSimnet(manifestPath?: string): Promise<Simnet>
 ```
 
-The `initSimnet` function takes the manifest path (`Clarinet.toml`) as an optional argument. By default, it'll look for a manifest in the current working directory.  
-It will often be the first function to call when using the SDK.
+The `initSimnet` function takes the manifest path (`Clarinet.toml`) as an optional argument. By default, it'll look for a manifest in the current working directory. It will often be the first function to call when using the SDK.
```

Comment:
```suggestion
The `initSimnet` function takes the manifest path (`Clarinet.toml`) as an optional argument. By default, it'll look for a manifest in the current working directory.
It will often be the first function to call when using the SDK.
```